lib/services/ngsi/entities-NGSI-v2.js

```diff
@@ -347,7 +347,7 @@ function isFloat(value) {
 function sendUpdateValueNgsi2(entityName, attributes, typeInformation, token, callback) {
```

Comment:
Maybe CNR should include something like this?

```
- Fix: avoid usage of newlines in logs
```

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1356/commits/0d81152d8cc0aa80f21c2ec2de2112ae60e1b83d